src/store-api/src/storage/types.rs

```diff
@@ -18,39 +18,27 @@
 /// as a sequence number.
 pub type SequenceNumber = u64;
 
-/// Operation type of the value to write to storage.
-///
-/// The enum values are stored in the SST files so don't change
-/// them if possible.
-#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
-pub enum OpType {
-    /// Delete operation.
-    Delete = 0,
-    /// Put operation.
-    Put = 1,
-}
-
-impl OpType {
-    /// Cast the [OpType] to u8.
-    #[inline]
-    pub fn as_u8(&self) -> u8 {
-        *self as u8
-    }
-
-    /// Minimal op type after casting to u8.
-    pub const fn min_type() -> OpType {
-        OpType::Delete
-    }
-}
-
-#[cfg(test)]
-mod tests {
-    use super::*;
-
-    #[test]
-    fn test_op_type() {
-        assert_eq!(0, OpType::Delete.as_u8());
-        assert_eq!(1, OpType::Put.as_u8());
-        assert_eq!(0, OpType::min_type().as_u8());
-    }
-}
+// impl OpType {
+//     /// Cast the [OpType] to u8.
+//     #[inline]
+//     pub fn as_u8(&self) -> u8 {
+//         *self as u8
+//     }
+//
+//     /// Minimal op type after casting to u8.
+//     pub const fn min_type() -> OpType {
+//         OpType::Delete
+//     }
+// }
+//
+// #[cfg(test)]
+// mod tests {
+//     use super::*;
+//
+//     #[test]
+//     fn test_op_type() {
+//         assert_eq!(0, OpType::Delete.as_u8());
+//         assert_eq!(1, OpType::Put.as_u8());
+//         assert_eq!(0, OpType::min_type().as_u8());
+//     }
+// }
```

Comment:
```suggestion
```

Reply:
Fixed in https://github.com/GreptimeTeam/greptimedb/pull/2170/commits/d9a2bba2349fc4e606112f2d6c9ea8ea64e48390